src/lib/log/logLevel.ts

```diff
@@ -9,6 +10,9 @@ import { formatLog } from './logUtils';
 const ERRORS_WEBHOOK = process.env.NEXT_PUBLIC_DISCORD_WEBHOOK_ERRORS;
 const originalFactory = _log.methodFactory;
 
+// requests per second = 35, timeUnit = 1sec
+const discordRateLimiter = RateLimit(30);
```

Comment:
Minor tweak, can we set this as an environment variable & default to 30 if not present?

Reply:
hm i think that is overkill, and it requires you to go check the env to see the true limit. we have lots of rate limits - it would be really hard to manage that for all of them. I dont see us needing to swap the rate limit so quickly